deluge_client/client.py

```diff
@@ -174,9 +179,14 @@ def _receive_response(self, deluge_version, partial_data=b''):
                 exception_msg = b', '.join(exception_msg)
```

Comment:
Won't this crash still on python 3 when decode_utf8 is on? I made a PR against this PR to increase the test coverage.

Reply:
That should do it, the tests pass too.

Reply:
Nice, looks good to me too.